torchvision/prototype/datapoints/_image.py

```diff
@@ -289,6 +289,10 @@ def gaussian_blur(self, kernel_size: List[int], sigma: Optional[List[float]] = N
         )
         return Image.wrap_like(self, output)
 
+    def normalize(self, mean: List[float], std: List[float], inplace: bool = False):
```

Comment:
Maybe one thing that we should do already is to remove

https://github.com/pytorch/vision/blob/c06d52b1c5f6aee36802661c3ebc6347b97cc59e/test/prototype_transforms_dispatcher_infos.py#L429

to enable automated testing for this.

Reply:
See https://github.com/pytorch/vision/pull/7113#discussion_r1082255402.